### DISCUSSION TITLE
Custom Attribute for jansPerson throwing error when setting as claim for a user.

### DISCUSSION THREAD

**Discussion by dhanan-np** [JanssenProject/jans#4761] [Q&A] [answered]
I am using MySql database, and created two attributes like **dummyAttribService1** and **dummyAttribService2**

When updating a user( **inum=342a7f8c-81cf-4571-8c36-bead137f70a7,ou=people,o=jans**) by adding these two attributes as claims. It filed to save.
Checked the logs and I found that the api in backed was trying to select the columns **dummyAttribService1** and **dummyAttribService2** from **jansPerson** and which was not there. 

Please guide me on this. I need to define 2-4 boolean attributes representing the service which user might subscribed to.

**Comment by ossdhaval** [comment, marked as answer]
Hi @dhanan-np Can you confirm if this is still an issue?

**Reply by dhanan-np** [reply]
Hello @ossdhaval, this was resolved. Thanks a lot.
I
